kotlin/render-multilayer-symbols/src/main/java/com/esri/arcgisruntime/sample/rendermultilayersymbols/MainActivity.kt

```diff
@@ -0,0 +1,567 @@
+        addComplexPolygon()
+        addComplexPolyline()
+    }
+
+    /**
+     * Creates the label graphics to be displayed above each category of symbol,
+     * and adds them to the graphics overlay.
+     */
+    private fun addTextGraphics() {
+        graphicsOverlay?.graphics?.addAll(
+            listOf(
+                Graphic(
+                    Point(-150.0, 50.0, SpatialReferences.getWgs84()),
+                    getTextSymbol("MultilayerPoint\nSimple Markers")
+                ),
+                Graphic(
+                    Point(-80.0, 50.0, SpatialReferences.getWgs84()),
+                    getTextSymbol("MultilayerPoint\nPicture Markers")
+                ),
+                Graphic(
+                    Point(0.0, 50.0, SpatialReferences.getWgs84()),
+                    getTextSymbol("Multilayer\nPolyline")
+                ),
+                Graphic(
+                    Point(65.0, 50.0, SpatialReferences.getWgs84()),
+                    getTextSymbol("Multilayer\nPolygon")
+                ),
+                Graphic(
+                    Point(130.0, 50.0, SpatialReferences.getWgs84()),
+                    getTextSymbol("Multilayer\nComplex Symbols")
+                )
+            )
+        )
+    }
+
+    /**
+     * @return the TextSymbol with the [text] to be displayed on the mao
+     */
+    private fun getTextSymbol(text: String): TextSymbol {
+        val textSymbol = TextSymbol(
+            20F,
+            text,
+            Color.BLACK,
+            TextSymbol.HorizontalAlignment.CENTER,
+            TextSymbol.VerticalAlignment.MIDDLE
+        )
+        // give the text symbol a white background
+        textSymbol.backgroundColor = Color.WHITE
+        return textSymbol
+    }
+
+    /**
+     * Create picture marker symbols from URI or bitmap
+     */
+    private fun addImageGraphics() {
+        // URI of image to display
+        val bluePinImageURI =
+            "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Recreation/FeatureServer/0/images/e82f744ebb069bb35b234b3fea46deae/blue_pin.png"
+        // load the PictureMarkerSymbolLayer using the image URI
+        val pictureMarkerFromUri =
+            PictureMarkerSymbolLayer(bluePinImageURI)
+        pictureMarkerFromUri.addDoneLoadingListener {
+            if (pictureMarkerFromUri.loadStatus == LoadStatus.LOADED) {
+                // add loaded layer to the map
+                addGraphicFromPictureMarkerSymbolLayer(pictureMarkerFromUri, 0.0)
+            } else {
+                showError("Picture marker symbol layer failed to load from URI: ${pictureMarkerFromUri.loadError.message}")
+            }
+        }
+        pictureMarkerFromUri.loadAsync()
+
+        // new thread to make network request to retrieve bitmap of the same image
+        Thread {
+            val bitmap =
+                BitmapFactory.decodeStream(URL(bluePinImageURI).openConnection().getInputStream())
+            runOnUiThread {
```

Comment:
In other samples we've bundled the image into the assets folder--could be a good alternative here possibly? Maybe a URL to something on the web is fine too. Just something to think about

https://github.com/Esri/arcgis-runtime-samples-android/tree/main/java/viewshed-geoelement/src/main/assets

Reply:
I'll move this to the assets folder!